### PR TITLE
frosh 2021 changes

### PR DESCRIPTION
**Pull request by JohnWu966**
Added new page for Frosh Week 2021.

Changed link on main website to link to new page.

Changed formatting on main site's 'CSSS Frosh Week' page to have proper floating

### CODE REVIEW

**Comment by modernNeo** [comment]
please validate the htmls with https://validator.w3.org/

File: csss-site/src/events/frosh/templates/frosh/frosh_week.html [hunk 1]
```diff
@@ -5,17 +5,24 @@
...
-{#    <p class="lead">Check out <a href="{{ URL_ROOT }}events/frosh/2020">The Official Frosh Week Site</a> for more info, and to register!</p>#}
-    <h3 class="title is-size-2">The RAMing Ceremony</h3><hr>
-    <p class="lead">Our most hallowed of rituals. It doesn't hurt, we promise.</p>
-    <h3 class="title is-size-2">Pac Macro</h3>
-    <hr><p class="lead">The Frosh Week classic: become Pacman himself among the streets of downtown Vancouver. Pacman and the ghosts chase each other amidst the crowds and traffic, all the while being directed by their team mates, stationed at Harbour Centre. There, they have a live overview of the positions of all players, projected on a large screen, and managed by custom CSSS software.</p>
-    <h3 class="title is-size-2">Midnight Madness</h3>
-    <hr>
-    <p class="lead">An all-night event in the ASB, where the four Applied Science schools (Comp Sci, Engineering, Software Systems, and Mechtronics) go head-to-head in various challenges, until they arrive at the conclusion that everyone knew already: Comp Sci is the best.</p>
+    <section class="section">
```
Comment (modernNeo) [inline csss-site/src/events/frosh/templates/frosh/frosh_week.html:8]: why did you change the whole file?

**Review by modernNeo — CHANGES_REQUESTED** [review]
please validate the htmls with https://validator.w3.org/

and where are the following files used?

- csss-site/src/events/frosh/static/frosh_static/2021/images/Temp_Logo.png
- csss-site/src/events/frosh/static/frosh_static/2021/images/background.png
- csss-site/src/events/frosh/static/frosh_static/2021/images/content-background.png
- csss-site/src/events/frosh/static/frosh_static/2021/images/content-background2.png
- csss-site/src/events/frosh/static/frosh_static/2021/images/flag-amber.png
- csss-site/src/events/frosh/static/frosh_static/2021/images/title.png
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/css/main.css
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/bootstrap.min.js
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery-2.2.4.min.js
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery-func.js
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery.easing.1.3.js
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/modernizr.custom.js
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/retina.js
- csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/smoothscroll.js

File: csss-site/src/events/frosh/templates/frosh/frosh_week.html (hunk 1 above)
Comment (JohnWu966) [inline csss-site/src/events/frosh/templates/frosh/frosh_week.html:8]: The current https://sfucsss.org/events/frosh/ doesn't have any margins, so the text content is being displayed from side to side.  The only actual change here is the addition of lines 8 and line 27 which surrounds the content in a section with proper margins. Every line in between is the exact same. Not sure why Git counts it as different code. 

tl;dr theres only one actual change here, which adds proper margins to the page. Every other detected change to the file is a false positive.

Comment (JohnWu966) [reply csss-site/src/events/frosh/templates/frosh/frosh_week.html:8]: The current https://sfucsss.org/events/frosh/ doesn't have any margins, so the text content is being displayed from one end of the screen to the other. The only actual change here is the addition of lines 8 and line 27 which surrounds the content in a section with proper margins. Every line in between is the exact same. Not sure why Git counts it as different code.

tl;dr theres only one actual change here, which adds proper margins to the page. Every other detected change to the file is a false positive.

**Comment by JohnWu966** [comment]
> please validate the htmls with https://validator.w3.org/
> 
> and where are the following files used?
> 
> * csss-site/src/events/frosh/static/frosh_static/2021/images/Temp_Logo.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/background.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/content-background.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/content-background2.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/flag-amber.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/title.png
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/css/main.css
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/bootstrap.min.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery-2.2.4.min.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery-func.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery.easing.1.3.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/modernizr.custom.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/retina.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/smoothscroll.js



> please validate the htmls with https://validator.w3.org/
> 
> and where are the following files used?
> 
> * csss-site/src/events/frosh/static/frosh_static/2021/images/Temp_Logo.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/background.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/content-background.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/content-background2.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/flag-amber.png
> * csss-site/src/events/frosh/static/frosh_static/2021/images/title.png
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/css/main.css
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/bootstrap.min.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery-2.2.4.min.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery-func.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/jquery.easing.1.3.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/modernizr.custom.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/retina.js
> * csss-site/src/events/frosh/static/frosh_static/2021/mainsite/js/smoothscroll.js

removed all unnecessary files

**Review by modernNeo — CHANGES_REQUESTED** [review]
did you miss

- frosh_static/2021/images/background.png
- frosh_static/2021/images/content-background.png

?

and it seems you still didnt validate.

**Comment by JohnWu966** [comment]
> did you miss
> 
> * frosh_static/2021/images/background.png
> * frosh_static/2021/images/content-background.png
> 
> ?
> 
> and it seems you still didnt validate.

Those two files are used. They're the stone and paper textures that are used for the background of the site. You can find them being referenced in the frosh_static/2021/frosh2021.css file.

As for the validation, the validator complains about a lot of things that I can't change. For example, it does not like the `{` and `}` used when calling for static images. The validator complains about every other frosh html file as well. It doesn't seem like there's any way to satisfy it without rewriting the entire website.

File: csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
+
+
+body{
```
Comment (modernNeo) [inline csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css:3]: I think I would prefer it if you put these 2 rules that have relative paths in the html itself and use the `static` django template filter to link them.

It is generally best practice to put the css rules in the css file, I make an exception for rule that use paths to other files, just cause it makes it easier to debug in pycharm.

**Comment by modernNeo** [comment]
> As for the validation, the validator complains about a lot of things that I can't change. For example, it does not like the `{` and `}` used when calling for static images. The validator complains about every other frosh html file as well. It doesn't seem like there's any way to satisfy it without rewriting the entire website.

I dont see any errors or warnings complaining about `{`

https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-387%2Fevents%2Ffrosh%2F2021%2Ffrosh

https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-387%2Fevents%2Ffrosh%2F2021

**Comment by JohnWu966** [comment]
here are the errors when I use the upload file option on my local copy of the same html file 

https://cdn.discordapp.com/attachments/484286781173530624/881056834767699968/unknown.png

**Comment by modernNeo** [comment]
> here are the errors when I use the upload file option on my local copy of the same html file
> 
> https://cdn.discordapp.com/attachments/484286781173530624/881056834767699968/unknown.png

you are supposed to upload the html that django renders for you.....not the html that has django markdown.

take advantage of the staging server to view the actual errors

**Comment by modernNeo** [comment]
just look at the links I showed you to get a sense of where the errors are.

**Comment by modernNeo** [comment]
I still see errors at both of these links

https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-387%2Fevents%2Ffrosh%2F2021%2Ffrosh

https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-387%2Fevents%2Ffrosh%2F2021

**Comment by modernNeo** [comment]
still seeing errors at https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-387%2Fevents%2Ffrosh%2F2021

File: csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css (hunk 2 above)
Comment (modernNeo) [reply csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css:3]: @JohnWu966 and you still need to deal with this I believe.

**Comment by modernNeo** [comment]
still seeing warnings at https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-387%2Fevents%2Ffrosh%2F2021

File: csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css (hunk 2 above)
Comment (JohnWu966) [reply csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css:3]: As far as I've know, its not possible. Linking an image using `static` requires two sets of quotation marks. Setting images on the background of a div is possible in the html file itself if you use in-line CSS, but that requires the use of an additional set of quotation marks. Unless there's a third type of quotation mark that I'm unaware of, then its not possible to use static images and in-line css in conjunction.

**Comment by JohnWu966** [comment]
> still seeing warnings at https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-387%2Fevents%2Ffrosh%2F2021

fixed

File: csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css (hunk 2 above)
Comment (modernNeo) [reply csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css:3]: it is possible

https://github.com/CSSS/csss-site/blob/master/csss-site/src/csss/templates/csss/header.html#L18-L28

Comment (modernNeo) [reply csss-site/src/events/frosh/static/frosh_static/2021/css/frosh2021.css:3]: you still need to remove the rules from the CSS

**Review by modernNeo — CHANGES_REQUESTED** [review]
